.gitlab-ci.yml

```diff
@@ -485,6 +485,15 @@ doc:stdlib:dune:
   extends: .dune-ci-template
   variables:
     DUNE_TARGET: stdlib-html
+  script:
+    # Warning: will rebuild Coq, need to fix Dune to know about glob files.
+    # c.f. https://github.com/coq/coq/issues/12699
+    - rm -rf _build
+    - set -e
+    - echo 'start:coq.test'
```

Comment:
Could we stop copying these echos around? They're just leftover from travis.

Reply:
We left them in place as a reminder to add them once gitlab support log folding, I wonder if that's already the case?

Reply:
https://docs.gitlab.com/ee/ci/pipelines/index.html#custom-collapsible-sections